lib/js/src/manager/lifecycle/_LifecycleManager.js

```diff
@@ -506,7 +506,7 @@ class _LifecycleManager {
     }
 }
 
-_LifecycleManager.MAX_RPC_VERSION = new Version(6, 0, 0);
+_LifecycleManager.MAX_RPC_VERSION = new Version(6, 2, 0);
```

Comment:
```suggestion
_LifecycleManager.MAX_RPC_VERSION = new Version(7, 0, 0);
```

Reply:
@SilvioDC-Ford fixed in https://github.com/smartdevicelink/sdl_javascript_suite/pull/264/commits/8035899de3a2b59eae39bf9806b90bdabd46cdc7